test/util/range.cpp

```diff
@@ -213,6 +213,21 @@ void test_pointers()
     BOOST_CHECK(bgr::at(r2, 5) == 5);
 }
 
+void test_const_pointers()
```

Comment:
With the Jamfile snippet above, the workflow is identical. The only visible difference for the user is the (automatic) build of the UTF framework once, before the compilation of the tests themselves.
I'll try to publish a branch later today to demonstrate that.